misc/install_initrd.sh

```diff
@@ -0,0 +1,245 @@
+#!/bin/bash
+
+ihelp()  {
+
+    # echo "$(basename $0) - installs hook and boot script to enable rapiddisk cache on boot volume."
+    # echo
+    echo "Usage:"
+    echo "$(basename "$0") --boot=<boot_partition> --size=<ramdisk_size> --kernel=<kernel_version> [--force]"
+    echo "$(basename "$0") --kernel=<kernel_version> --uninstall [--force]"
+    echo ""
+    echo "Where:"
+    echo ""
+    echo "<boot_partition> is the device mounted as '/' as in /etc/fstab in the /dev/xxxn format"
```

Comment:
There may be a bit of confusion for the user as sometimes the "boot partition" may reside as /boot in its own partition. Just a suggestion to relabel "boot_partition" here and above, alongside the --boot arg as "root_partition" and --root. Thoughts?

Reply:
Using the "root" word is way better!